app/src/main/res/values-night/color_palette.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <color name="oppia_primary_color">@color/colorPrimary_DarkMode</color>
```

Comment:
The color code `colorPrimary_DarkMode` is not properly named. All color codes should be `snake_case` format.

Reply:
Done.